maui-toolkit/TabView/CenterButton-Customization.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: CenterButton Customization in .NET MAUI Tab View (SfTabView) | Syncfusion®
+description: Learn here all about CenterButton Customization in .NET MAUI Tab View control and more.
```

Comment:
Learn here all about Center Button Customization in .NET MAUI Tab View control and more.